code/wdb/wdbstandard.q

```diff
@@ -0,0 +1,6 @@
+\d .wdb
+startup:{[f]
```

Comment:
I think you'll need two args here to prevent immediate execution when creating the projection

```suggestion
startup:{[f;x]
```